### DISCUSSION TITLE
Handlebars template file examples ?

### DISCUSSION THREAD

**Discussion by doctorfree** [farling42/obsidian-import-json#29] [Q&A] [answered]
Hi, I am trying to convert JSON to Markdown. I stumbled across this Obsidian plugin and it looks like it may be just what I need. I'm also trying to use Pandoc. However, with Pandoc it appears I need to learn how to write a Lua filter and with this Obsidian plugin it appears I need to learn how to write a Handlebars template.

Are there any example Handlebars templates that illustrate how to convert JSON to Markdown?

Thanks for any info/pointers. I just discovered this plugin yesterday so I am a complete noob. Off to dive back into the doc.

**Comment by farling42** [comment, marked as answer]
I think the readme file has links to the basic handlebars Web site as well as a separate link to the handlebars helpers.
